coral-trino/src/main/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverter.java

```diff
@@ -94,6 +94,10 @@ protected RelNode standardizeRel(RelNode relNode) {
     return relNode;
   }
 
+  private static String standardizeSql(String sql) {
+    return sql.toUpperCase().replaceAll("`", "\"");
+  }
+
   private static String trimParenthesis(String value) {
     String str = value.trim();
     if (str.startsWith("(") && str.endsWith(")")) {
```

Comment:
I think it's better to combine these 2 methods together in one method `standardizeTrinoSql` and add comment of the reason why we need to standardize it.